scripts/gh.sh

```diff
@@ -1,20 +1,27 @@
 #!/usr/bin/env bash
 SERVICE=$1
```

Comment:
Did you check why there is no comment in any PR for now ?

Reply:
Should be resolved now